CHANGELOG.md

```diff
@@ -4,6 +4,184 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.48.0] - 2024-05-29
+
+## 🚀 Features
+
+### Add support for studio trace id in selectors and document context entry ([Issue #3803](https://github.com/apollographql/router/issues/3803)), ([Issue #5172](https://github.com/apollographql/router/issues/5172))
+
+Add support for a new trace ID selector kind, the `apollo` trace ID, which represents the trace ID on [Apollo GraphOS Studio](https://studio.apollographql.com/). 
+
+An example configuration using `trace_id: apollo`:
+
+```yaml
+telemetry:
+  instrumentation:
+    spans:
+      router:
+        "studio.trace.id":
+            trace_id:: apollo
+```
+
+Also, router documentation has been updated with constants that are available in Rhai.
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/5189
+
+### Add ability for router to deal with query plans with contextual rewrites ([PR #5097](https://github.com/apollographql/router/pull/5097))
+
+Adds the ability for the router to execute query plans with context rewrites on them. These are generated by the @fromContext directive and are used to map a Value in the collected data JSON onto a variable which will in turn be used as an argument to a field resolver.
+
+⚠️ This ships with a new version of federation, 2.8.0, which means distributed caches will be repopulated.
+
+By [@clenfest](https://github.com/clenfest) in https://github.com/apollographql/router/pull/5097
+
+## 🐛 Fixes
+
+### Submit new metrics to Apollo ([PR #5270](https://github.com/apollographql/router/pull/5270))
```

Comment:
This isn't a fix. If we merge it, fine, but otherwise maybe it's a chore?

Reply:
I put the other one under config, but we can move both?